### PR TITLE
minor copy edits

### PR DESCRIPTION
**Pull request by labarba**
https://github.com/openjournals/joss-reviews/issues/1759

### CODE REVIEW

**Comment by MuellerSeb** [comment]
I will update it myself, so you don't have to do it.

File: paper/paper.md [hunk 1]
```diff
@@ -49,41 +49,40 @@ d_{3}^{\left(-2\right)} & d_{3}^{\left(-1\right)} & d_{3} & d_{3}^{\left(1\right
...
 
-Here the $d_i$ are the diagonal entries and the $d_i^{(j)}$ represent the $j$-th minor diagonal.
+Here, $d_i$ are the diagonal entries and $d_i^{(j)}$ represent the $j$-th minor diagonal.
 
-Recently @askar presented two algorithms ``PTRANS-I`` and ``PTRANS-II``,
-which solve the linear equation systems for $X$ by transformation to
-a triangular matrix and subsequent backward resp. forward substitution.
+Recently @askar presented two algorithms, ``PTRANS-I`` and ``PTRANS-II``,
+that solve the linear systems of equations for $X$ by transformation to
+a triangular matrix and subsequent backward and forward substitution.
```
[the remark this answers is not in the document]
Comment (labarba) [reply paper/paper.md]: I see. The sentence is still hard to parse, given the distance between mentioning PTRANS I and II, and the "respectively." I will make a suggestion on the document, directly.
